korman/properties/modifiers/water.py

```diff
@@ -471,3 +471,38 @@ class PlasmaWaveTexState(PlasmaWaveState, PlasmaModifierProperties):
     def export(self, exporter, bo, so):
         waveset = exporter.mgr.find_create_object(plWaveSet7, name=bo.name, so=so)
         self.convert_wavestate(waveset.state.texState)
+
+
+class PlasmaBuoyObject(bpy.types.PropertyGroup):
+    buoy_object = PointerProperty(name="Buoy Object",
+                                  description="Object that float on water",
+                                  options=set(),
+                                  type=bpy.types.Object,
+                                  poll=idprops.poll_dynamic_objects)
```

Comment:
While this is correct from an implementation standpoint, I feel like it's putting too much of a requirement on the user on knowing how buoys work under the hood. Instead, I think we should call the physics exporter in the export fxn and place the resulting physical in the dynamic group... `exporter.physics.generate_physical(bo, so, member_group="kGroupDynamic")` This poll fxn should probably just check for mesh objects. Further, we should alert in the UI (and maybe the log too?) if the collision mod is on with dynamic unticked. Alternatively, add instrumentation to the physics exporter to force buoys dynamic.